examples/powercli/sdrs-recommendation-webexteams/README.md

```diff
@@ -0,0 +1,93 @@
+# vSphere storage DRS recommendation Notification
+
+## Description
+
+This function demonstrates using PowerCLI to a notification to Cisco WebEx Teams when a new storage DRS recommendation is available.
+
+## Consume Function Instruction
+
+Step 1 - Clone repo
+
+```
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance
+cd vcenter-event-broker-appliance/examples/powercli/sdrs-recommendation-webexteams
+git checkout master
+```
+
+Step 2 - Update `stack.yml` and `vc-sdrs-config.json` with your environment information
+
+> **Note:** you will need to create a bot on Webex Teams and copy the token. You will also need to add the bot to a space and get the ID of that space.  
+You can get the room ID by listing all the rooms on <https://developer.webex.com/docs/api/v1/rooms/list-rooms> .  
+
+
+Step 3 - Login to the OpenFaaS gateway on vCenter Event Broker Appliance
+
+```
+VEBA_GATEWAY=https://veba.primp-industries.com
+export OPENFAAS_URL=${VEBA_GATEWAY}
+
+faas-cli login --username admin --password-stdin --tls-no-verify
```

Comment:
Just to verify: `--password-stdin` is given (also below) but I don't see it being passed in via standard input?

Reply:
To be honest, I started and adapted the readme file from an existing example function. I did see this bit in nearly all of them. But now that I look at it, this could use some clarification.

I'll update this bit.

Reply:
Were you able to make progress on this change?

Reply:
I've updated the readme file in commit [c31e587](https://github.com/vmware-samples/vcenter-event-broker-appliance/commit/c31e587cfbeadd83bf4937b49c3b278a556e64ce) to clarify this.